### DISCUSSION TITLE
Error: The \"readableStream\" argument must be an instance of ReadableStream. Received an instance of PassThrough

### DISCUSSION THREAD

**Discussion by ilya007500** [janhq/jan#3144] [Get Help] [answered]
Starting from version 0.5.0 when using the Local API Server on request: http://127.0.0.1:1337/v1/chat/completions an error is displayed in the response:

"statusCode": 404,
"code": "ERR_INVALID_ARG_TYPE",
"error": "Not Found",
""message": "The \"readable Stream\" argument must be an instance of Readable Stream. Received an instance of Pass Through"

Has anyone ever made such a mistake? Can you tell me how to solve it?

**Comment by ux-han** [comment, marked as answer]
This issue is solved here https://github.com/janhq/jan/issues/3163
